promql/engine_test.go

```diff
@@ -2124,3 +2150,8 @@ func TestEngineOptsValidation(t *testing.T) {
 		}
 	}
 }
+
+// TestFuncTimestampWithAtModifier tests for https://github.com/prometheus/prometheus/issues/8433.
+func TestFuncTimestampWithAtModifier(t *testing.T) {
+
+}
```

Comment:
Post merge review for myself, this empty test got committed while I moved the test case to existing tests. To be removed (I will do it tomorrow).